client/state/themes/hooks/use-is-theme-allowed-on-site.ts

```diff
@@ -6,13 +6,26 @@ import { getThemeTierForTheme } from 'calypso/state/themes/selectors';
 export function useIsThemeAllowedOnSite( siteId: number | null, themeId: string ) {
 	const isThemeAllowed = useSelector( ( state ) => {
 		const themeTier = getThemeTierForTheme( state, themeId );
-		return themeTier?.feature ? siteHasFeature( state, siteId, themeTier.feature ) : true;
+		const features = themeTier?.featureList ?? [ themeTier?.feature ] ?? [ null ];
+
+		return features.some(
+			( feature: string | null | undefined ) =>
+				! feature || siteHasFeature( state, siteId, feature )
+		);
```

Comment:
```suggestion
		if( ! themeTier?.featureList ) return true;

		return siteHasFeature( state, siteId, themeTier?.feature )
```

Since there's only one array element, I think the code is easier to read if we avoid using the `some` method.

Reply:
There isn't always one element in the array, there can be n elements.

Reply:
Nevermind, I thought it was a ternary operator instead of a nullish coalescing one 😅